### ISSUE TITLE
Support of integer fields

### ISSUE THREAD

**Issue by dejandular** [BESDev/Celedon-Autonumber-RTM#15]
Have you ever considered supporting auto numbering on integer fields? Currently only string fields are supported. Of course if the field is integer, then prefix, suffix and number formatting is ignored.

**Comment by tsharp** [comment]
It is an interesting proposition - would the use case be an auto-generated alternative integer key for the record?

**Comment by dejandular** [comment]
Yes that is the idea. Maybe you can allow generating the auto-generated numbers on integer fields if the autonumber rule does not have any prefix or suffix.
